rasa/core/schemas/domain.yml

```diff
@@ -27,6 +27,39 @@ mapping:
   responses:
```

Comment:
@degiz since I copy-pasted the definition of the `responses` key in both schemas, I thought about maybe defining the response schema in `rasa/utils/schemas/responses.yml` and [including](https://pykwalify.readthedocs.io/en/latest/partial-schemas.html#include) it in both domain and nlu. Have you thought about this?

Reply:
The reason `responses` are in the nlu schema is to allow users to keep their training data in one file, but each individual reader would use it's own schema.
The approach is not perfect though 🤦 I'm still thinking how to do it properly and even have an items for that [here](https://github.com/orgs/RasaHQ/projects/30)

Reply:
I'm not saying we should keep everything in one schema. I'm simply proposing to remove the schema definition duplication, by using the [inclusion feature of `pykwalify`](https://pykwalify.readthedocs.io/en/latest/partial-schemas.html#include), that allows us to import a schema definition inside another schema definition. Does that make sense?

Reply:
@degiz check this out: dba1ebbe57a WDYT?